src/haz3lcore/statics/ConstructorMap.re

```diff
@@ -99,3 +99,12 @@ let rec is_ground = (is_ground_value: 'a => bool, map: t('a)): bool =>
   | [(_, head), ...tail] =>
     is_ground_value(head) && tail |> is_ground(is_ground_value)
   };
+
+let nth = (ctr: Constructor.t, map: t('a)): option(int) => {
```

Comment:
minor: by convention in OCaml, functions like this would have the map as the first argument.